requirements.txt

```diff
@@ -1,6 +1,7 @@
 adal==0.4.3
 applicationinsights==0.10.0
 argcomplete==1.8.0
+azure_devtools==0.1.0
```

Comment:
- Never use an underscore in published package name ([see this pip issue for instance](https://github.com/pypa/pip/issues/3212))
- I have no opinion on this file or `scripts/setup.py`, since I don't know where the later is actually called